integrations/arize.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: integration
+name: Arize AI
+description: Monitor and trace your Haystack pipelines Arize and Phoenix
+authors:
+    - name: Arize AI
+      socials:
+        github: Arize-ai
+        twitter: arizeai
```

Comment:
I think it might make sense to have this be https://twitter.com/ArizePhoenix as we tend to push more developer content through our OSS twitter.